sycl/doc/GettingStartedAlveo.md

```diff
@@ -0,0 +1,449 @@
+
+If you want to use a real board, follow the recipe "Generating the
+xbutil flash Command" from
+https://www.xilinx.com/html_docs/accelerator_cards/alveo_doc/ftt1547585535561.html
+about how to concretely generate the exact flashing command.
+
+Typically you run:
+```bash
+sudo /opt/xilinx/xrt/bin/xbutil flash scan
+XBFLASH -- Xilinx Card Flash Utility
+Card [0]
+	Card BDF:		0000:04:00.0
+	Card type:		u200
+	Flash type:		SPI
+	Shell running on FPGA:
+		xilinx_u200_GOLDEN_2,[SC=1.8]
+	Shell package installed in system:	
+		xilinx_u200_xdma_201830_1,[TS=0x000000005bece8e1],[SC=3.1]
+```
+to get the information about the installed shell and you translate this
+into a flashing command according to the parameters above:
+```bash
+sudo /opt/xilinx/xrt/bin/xbutil flash -a xilinx_u200_xdma_201830_1 -t 0x000000005bece8e1 -d 0
+XBFLASH -- Xilinx Card Flash Utility
+Probing card[0]: Shell on FPGA needs updating
+Shell on below card(s) will be updated:
+Card [0]
+Are you sure you wish to proceed? [y/n]
+y
+Updating SC firmware on card[0]
+INFO: found 5 sections
+..............................
+INFO: Loading new firmware on SC
+.
+Updating shell on card[0]
+INFO: ***Found 701 ELA Records
+Idcode byte[0] ff
+Idcode byte[1] 20
+Idcode byte[2] bb
+Idcode byte[3] 21
+Idcode byte[4] 10
+Enabled bitstream guard. Bitstream will not be loaded until flashing is finished.
+Erasing flash...................................
+Programming flash...................................
+Cleared bitstream guard. Bitstream now active.
+1 Card(s) flashed successfully.
+Cold reboot machine to load the new image on FPGA.
+```
+
+Unfortunately you need to reboot the machine to have the new shell
+loaded inside the FPGA.
+
+Then you can check with a pre-compiled FPGA program provided by the
+shell that the board is working correctly with:
+```bash
+sudo /opt/xilinx/xrt/bin/xbutil validate
+INFO: Found 1 cards
+
+INFO: Validating card[0]: xilinx_u200_xdma_201830_1
+INFO: Checking PCIE link status: PASSED
+INFO: Starting verify kernel test: 
+INFO: verify kernel test PASSED
+INFO: Starting DMA test
+Host -> PCIe -> FPGA write bandwidth = 11127.5 MB/s
+Host <- PCIe <- FPGA read bandwidth = 12147.9 MB/s
+INFO: DMA test PASSED
+INFO: Starting DDR bandwidth test: ..........
+Maximum throughput: 47665.777344 MB/s
+INFO: DDR bandwidth test PASSED
+INFO: Starting P2P test
+P2P BAR is not enabled. Skipping validation
+INFO: P2P test PASSED
+INFO: Starting M2M test
+bank0 -> bank1 M2M bandwidth: 12089.7 MB/s	
+bank0 -> bank2 M2M bandwidth: 12064.7 MB/s	
+bank0 -> bank3 M2M bandwidth: 12084 MB/s	
+bank1 -> bank2 M2M bandwidth: 12064.1 MB/s	
+bank1 -> bank3 M2M bandwidth: 12066.9 MB/s	
+bank2 -> bank3 M2M bandwidth: 12125.8 MB/s	
+INFO: M2M test PASSED
+INFO: Card[0] validated successfully.
+
+INFO: All cards validated successfully.
+```
+
+It looks there is a problem in the Linux `udev` configuration not
+making the DRM driver accessible for some `other` users. A crude
```

Comment:
"configuration not making the DRM driver accessible " -> "configuration making the DRM driver inaccessible ", reads a little easier i think